Platforms/QemuQ35Pkg/CpuInfoDxe/CpuInfoDxe.c

```diff
@@ -0,0 +1,129 @@
+/** @file CpuInfoDxe.c
+
+    Copyright (c) Microsoft Corporation. All rights reserved.
+
+    Driver that prints CPU branding information
+**/
+
+#include <Uefi.h>
+#include <Library/UefiLib.h>
+#include <Library/DebugLib.h>
+#include <Library/UefiDriverEntryPoint.h>
+#include <Library/UefiBootServicesTableLib.h>
+#include <Library/BaseLib.h>
+#include <Library/PrintLib.h>
+#include <Library/UefiCpuLib.h>
+
+typedef union {
+  ///
+  /// Individual bit fields
+  ///
+  struct {
+    UINT32    SteppingId       : 4; ///< [Bits   3:0] Stepping ID
+    UINT32    Model            : 4; ///< [Bits   7:4] Model
+    UINT32    FamilyId         : 4; ///< [Bits  11:8] Family
+    UINT32    ProcessorType    : 2; ///< [Bits 13:12] Processor Type
+    UINT32    Reserved1        : 2; ///< [Bits 15:14] Reserved
+    UINT32    ExtendedModelId  : 4; ///< [Bits 19:16] Extended Model ID
+    UINT32    ExtendedFamilyId : 8; ///< [Bits 27:20] Extended Family ID
+    UINT32    Reserved2        : 4; ///< Reserved
+  } Bits;
+  ///
+  /// All bit fields as a 32-bit value
+  ///
+  UINT32    Uint32;
+} CPUID_VERSION_INFO_EAX;

```

Comment:
Was there a particular reason not to use `CPUID_VERSION_INFO_EAX` in [`Register/Intel/Cpuid.h`](https://github.com/microsoft/mu_basecore/blob/release/202302/MdePkg/Include/Register/Intel/Cpuid.h#L83-L105)?